Cargo.toml

```diff
@@ -9,7 +9,7 @@ crate-type = ["cdylib"]
 path = "extensions/route_map/lib.rs"
 
 [dependencies]
-pyo3 = { version = "0.16.5" }
+pyo3 = { version = "*" }
```

Comment:
This tends to be pretty frowned upon in rust, it's usually better to specify a version of dependencies.